packit_service/worker/reporting.py

```diff
@@ -254,12 +244,20 @@ def report_status_by_comment(
     def get_statuses(self):
         self.project_with_commit.get_commit_statuses(commit=self.commit_sha)
 
-    def _has_identical_comment(self, body: str, mode: DuplicateCheckMode) -> bool:
-        """Checks if the body is the same as the last or any comment based on mode."""
+    def _has_identical_comment(
+        self, body: str, mode: DuplicateCheckMode, commit: Optional[bool] = False
```

Comment:
nitpick: I'd rename `commit` to `check_commit` to make it clear that is is not some kind of git commit object